_data/projects/TakeMeThere.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: Take Me There
+desc: A URL Shortening service powered by Google Apps Script
+site: https://tmt.pw
+tags:
+- google-apps-script
+- url-shortener
+- github-pages
+- open-source
+upforgrabs:
+  name: help needed with the line items in bucket list
+  link: https://github.com/schoraria911/gas-url-shortener#bucket-list
```

Comment:
@schoraria911 This should be the label name such as `good first issue`
link should be like https://github.com/schoraria911/gas-url-shortener/labels/good%20first%20issue

Reply:
@ritwik12 - thanks for pointing this out. Looks like this was a dumb mistake on my part. I have created my _bucket list_ items as issues under the `good first use` label and the link that you've proposed now has a list of 3 items there.

Also, I assumed my original repo for this file was no longer needed and so I deleted it (my bad! new to actually having done this) but would it be possible for you update my file on the **up-for-grabs** repo directly?

If not, do please let me know and so I can close this pull request and initiate a new one. Thanks.